app/email/notify-send.js

```diff
@@ -107,18 +107,22 @@ export const sendFarmerApplicationEmail = async (data, blob) => {
     sbi: data.sbi
   }
 
-  const emailAddress = data.email
+  const { email, orgEmail } = data
+
   const emailTemplateId = data.userType === NEW_USER ? templateIdFarmerApplicationGenerationNewUser : templateIdFarmerApplicationGenerationExistingUser
   let successFullySent = true
 
+  // send to RPA
   sendCarbonCopy(personalisation, data.reference, emailTemplateId)
 
-  if (data?.orgEmail) {
-    successFullySent = await sendEmail(data.orgEmail, personalisation, data.reference, emailTemplateId)
+  // send to orgEmail if present
```

Comment:
is it needed?

Reply:
The comment? I'd recommend we get rid shortly, when this gets refactored to cleanse the notify/SFD switchability